kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,21 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must create the required custom resources (CRs) and complete the Operator installation.
+
+NOTE: {KubeFedProductName} can be deployed as either a namespace-scoped or a cluster-scoped deployment, but not both, as this will cause conflicts when both deployments attempt to manage the same set of resources.
+
+include::modules/creating-kubefed-CRs.adoc[leveloffset=+1]
+
+== Removing previous versions of the {KubeFedProductName} Operator
+The {KubeFedProductName} operator does not currently support automatic upgrades. You must manually remove any existing versions of the operator before you install the latest version.
```

Comment:
I do not think this will be true in 4.2, so I think we want to strike this language around automatic upgrades.